i18n/de_DE.csv

```diff
@@ -158,12 +158,12 @@
 "Magento Enterprise Edition version","Magento Enterprise Edition-Version"
 "Quick reference guide","Kurzanleitung"
 "Log in to the ","Melden Sie sich an "
-"Buckaroo Payment Plaza","Buckaroo Payment Plaza"
+"Buckaroo Buckaroo Plaza","Buckaroo Buckaroo Plaza"
 " to collect the following security settings:"," um die folgenden Sicherheitseinstellungen zu sammeln:"
 "Support","Die Unterstützung"
 "This extension is developed by Buckaroo.","Diese Erweiterung wird von Buckaroo entwickelt."
 "Extension basic configuration and account information","Basiskonfiguration der Erweiterung und Kontoinformationen"
-"Buckaroo Payment Plaza:","Buckaroo Payment Plaza:"
+"Buckaroo Buckaroo Plaza:","Buckaroo Buckaroo Plaza:"
```

Comment:
same, sry

Reply:
**This should be Buckaroo Plaza:**
"Buckaroo Plaza:","Buckaroo Plaza:"